devel/convertPrimatives.hpp

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: BSL-1.0
```

Comment:
If you happen to add to any particular file, add your john hancock "// Copyright (c) 2017-2017 Coburn Wightman" just like convert.cpp. Its the same license, but any number of copyright holders is fine.